src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/Collect.txt

```diff
@@ -36,3 +36,9 @@ Table({Value:1},{Value:2},{Value:3},{Value:99},{Value:88},{Value:77},{Value:77},
 
 >> Collect(Table({x:1,y:2}), {x: 3}).y 
 Blank()
+
+>> Collect([1,2,3],{Value:"200"}).Value
```

Comment:
> Collect([1,2,3],{Value:"200"}).Value [](http://example.com/codeflow?start=2&length=37)

Can we add test, where value can't really be coerced?
e,g, `Collect([1,2,3],{Value:"run time error"}).Value`

Reply:
Similar thing for Patch

Reply:
Added.